builder/proxmox/common/config.go

```diff
@@ -58,12 +58,20 @@ type Config struct {
 	CloudInit            bool   `mapstructure:"cloud_init"`
 	CloudInitStoragePool string `mapstructure:"cloud_init_storage_pool"`
 
-	shouldUploadISO bool
-
 	AdditionalISOFiles []storageConfig `mapstructure:"additional_iso_files"`
 	VMInterface        string          `mapstructure:"vm_interface"`
 
-	Ctx interpolate.Context `mapstructure-to-hcl2:"skip"`
+	Ctx interpolate.Context `mapstructure-to-hcl2:",skip"`
+}
+
+type storageConfig struct {
+	common.ISOConfig `mapstructure:",squash"`
+	Device           string `mapstructure:"device"`
+	ISOFile          string `mapstructure:"iso_file"`
+	ISOStoragePool   string `mapstructure:"iso_storage_pool"`
+	Unmount          bool   `mapstructure:"unmount"`
+	ShouldUploadISO  bool
+	DownloadPathKey  string
```

Comment:
Was this meant to be moved into common? It should only be relevant for the iso builder, I think?

Reply:
That particular change was done by @azr so I assumed it was wanted; I have no strong opinions one way or the other.

Reply:
This `storageConfig` was created in https://github.com/hashicorp/packer/pull/9653 and should be also used to mount isos, I thought that was an option for `proxmox-clone` too so I brought it over.

Reply:
I'm _fairly_ sure we only use that in the iso builder, right @featheredtoast? The clone builder doesn't do any mounting at all?

Reply:
Yeah that's correct - can re-update that one :)

Reply:
After reviewing the feature, I believe @azr is correct here - while I don't have a use for it (yet), the additional ISO files option should be made available to proxmox-clone as well. From my understanding #9653 allows ISOs to be mounted after VMs are created, so they appear on first boot.

It's possible a packer config would want an ISO to be mounted after cloning an existing image, so I'd keep it.

Reply:
*Though this does mean that upload additional ISOs should probably be made available in common

Reply:
Alright, sounds good! :+1: